pkg/kubectl/cmd/set/set_image.go

```diff
@@ -137,10 +137,12 @@ func (o *ImageOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, args []st
 		return err
 	}
 
+	includeUninitialized := cmdutil.GetFlagBool(cmd, "include-uninitialized")
```

Comment:
See my comment at  https://github.com/kubernetes/kubernetes/issues/49035#issuecomment-322322225 I think `set` works with individual resources (and doesn't work in batch mode) so it should not have this flag.

ditto for other `set_*.go` changes.